git/remote.py

```diff
@@ -629,7 +634,7 @@ def _get_fetch_info_from_stderr(self, proc, progress):
         fetch_info_lines = list()
         # Basically we want all fetch info lines which appear to be in regular form, and thus have a
         # command character. Everything else we ignore,
-        cmds = set(PushInfo._flag_map.keys()) & set(FetchInfo._flag_map.keys())
```

Comment:
Maybe I make a mistake because I may not understand the project deeply.

I make this change for 3 reasons:

1. In the project before this commit, there is no difference between `set(PushInfo._flag_map.keys()) & set(FetchInfo._flag_map.keys())` and `set(FetchInfo._flag_map.keys())`.
`PushInfo._flag_map.keys() == ['X', '-', '*', '+', ' ', '=', '!']` and `FetchInfo._flag_map.keys() == ['!', '+', '-', '*', '=', ' ']`, and these two `_flag_map` are considered unchangable.

2. It seems that there is no need to make an intersection between Push and Fetch.
The only two usages of this function is `fetch` and `pull`. As the function (named `_get_fetch_info_from_stderr`) has the word `fetch` in its name, its duty is to *parse data from **fetch** messages*, and thus should only use `FetchInfo._flags_map`.
So, I think use `set(FetchInfo._flag_map.keys())` is just the right way.

3. Push command has no 't'.
Accoding to the documents of [git-fetch](https://github.com/git/git/blob/2cb040baa6fcb8a6314a54933cbcb4d3fcb60401/Documentation/git-fetch.txt#L120) and [git-push](https://github.com/git/git/blob/2cb040baa6fcb8a6314a54933cbcb4d3fcb60401/Documentation/git-push.txt#L321), fetch has more command than push now, make an intersection will make this function loss some types of records.

That's way I think I could and have to make this change.